examples/glue/run_glue_trainer.py

```diff
@@ -330,8 +368,30 @@ def train_finetune(
 
     trainer.fit()
     print(f'\nFINISHED TRAINING TASK {task.upper()}\n')
+
+    cpu_metrics = DeviceCPU().batch_to_device(trainer.state.current_metrics)
+    # convert the Torch tensors to primitives so they can be sent over multiprocessing
+    cpu_metrics = _convert_torch_tensor_to_primitives(cpu_metrics)
```

Comment:
Please add a `trainer.close()` call before this function exits

Reply:
Done!